README.md

```diff
@@ -1,64 +1,375 @@
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+---
+services: active-directory
+platforms: java
+author: ramya25
+level: 300
+client: Java web app
+service: Java Web API
+endpoint: Microsoft identity platform
+page_type: sample
+languages:
+  - java  
+products:
+  - azure
+  - azure-active-directory
+  - java
+  - office-ms-graph
+description: "This sample demonstrates a Java web app application calling a Java Web API that is secured using Azure Active Directory using the On-Behalf-Of flow"
+---
+# A Java Web API that calls another web API with the Microsoft identity platform using the On-Behalf-Of flow
+
+## About this sample
+
+### Overview
+
+This sample demonstrates a Java web application calling a downstream Web API, which in turn calls the [Microsoft Graph](https://graph.microsoft.com) using an [access token](https://docs.microsoft.com/en-us/azure/active-directory/develop/access-tokens) obtained using the [on-behalf-of](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow) flow. All these are secured using the [Microsoft identity platform (formerly Azure Active Directory for developers)](https://docs.microsoft.com/en-us/azure/active-directory/develop/).
+
+1. The Java web application uses the [Microsoft Authentication Library for Java (MSAL4J)](https://github.com/AzureAD/microsoft-authentication-library-for-java) to obtain an Access token the Microsoft identity platform for the authenticated user.
+2. The access token is then used as a bearer token to authorize the caller in the Java Web API and then subsequently for Microsoft Graph API.
+
+The flow is as follows:
+
+1. Sign-in the user in the client(web) application.
+1. Acquire an access token token for the Java Web API and call it.
+1. The Java Web API authorizes the caller and then calls another downstream Web API ([The Microsoft Graph](https://graph.microsoft.com)) after obtaining another [access token](https://docs.microsoft.com/en-us/azure/active-directory/develop/access-tokens) using the on-behalf-of flow.
+
+### Scenario
+
+This sample shows how to build a Java web app that uses the [OpenId Connect](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-v2-protocols) protocol to sign in and sign out an user and to get an access token for another Web API including for the Microsoft Graph using the MSAL4J library. It also shows how to use On-Behalf-Of flow to acquire another access token from a Wen API to call another API.
+For more information about how the protocols work in this scenario and other scenarios, see [Authentication Scenarios for Azure AD](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-authentication-scenarios),[Microsoft identity platform and OAuth 2.0 On-Behalf-Of flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow).
+
```

Comment:
This paragraph seems to be a repetition of what is explained very well in the overview section. I suggest we remove this paragraph. It is redundant and adding more text to be read.